build/yamls/base/controller.yml

```diff
@@ -91,6 +91,21 @@ webhooks:
   admissionReviewVersions: ["v1", "v1beta1"]
   sideEffects: None
   timeoutSeconds: 5
+- name: "anpvalidator.antrea.tanzu.vmware.com"
+  clientConfig:
+    service:
+      name: "antrea"
+      namespace: "kube-system"
+      path: "/validate/anp"
```

Comment:
not related to this PR, but should the above validator for cnp should be acnp to keep align?

Reply:
will do it with this PR.. thanks